packages/assert/README.md

```diff
@@ -3,8 +3,38 @@ assert
 
```

Comment:
Let's include a main header here with the package name like this: 

# @0xproject/assert

Package description directly underneath here